src/utils/components/TextInput.vue

```diff
@@ -62,6 +62,14 @@ const props = defineProps({
         type: [Boolean, null],
         default: null,
     },
+    /**
+     * Styling parameter which describes whether there is another element to the right (e.g another
+     * button) to determine the rounding of corners
+     */
+    isLastElement: {
+        type: Boolean,
+        default: true,
+    },
```

Comment:
See comment about slots, instead of this boolean input if you use slots you can make this a data and check the number of slots.